dev/build.image/build.gradle

```diff
@@ -106,7 +106,7 @@ File mavenDir = new File(project.buildDir, 'maven')
 class PackageLibertyWithFeatures extends DefaultTask {
     Closure withFeatures
     Boolean completeFeatureList
-    Boolean packageServerConflict
+    String packageServerConflict
```

Comment:
Some doc might be useful, what are the valid values?

Reply:
comments added to packageServerConflict